views/view_messages.erb

```diff
@@ -0,0 +1 @@
+<%= @messages %>
```

Comment:
over here you can just call message.chit in an each loop to view just the chits.